lib/structures/AutocompleteInteraction.js

```diff
@@ -32,18 +34,19 @@ class AutocompleteInteraction extends Interaction {
       id: data.channel_id,
     };
 
+    this.context = data.context;
     this.data = data.data;
 
     if (data.guild_id !== undefined) {
       this.guildID = data.guild_id;
     }
 
     if (data.member !== undefined) {
+      data.member.id = data.member.user.id;
       if (this.channel.guild) {
-        data.member.id = data.member.user.id;
         this.member = this.channel.guild.members.update(data.member, this.channel.guild);
       } else {
-        const guild = this._client.guilds.get(data.guild_id);
+        const guild = this._client.guilds.get(data.guild_id) || new Guild(data.guild, this._client);
```

Comment:
There's not enough data in the structure to warrant creating a whole new instance of Guild here. We can probably just get away with assigning the structure directly here